homeassistant/components/sensor/ebox.py

```diff
@@ -18,9 +18,9 @@
     CONF_USERNAME, CONF_PASSWORD,
     CONF_NAME, CONF_MONITORED_VARIABLES)
 from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
 
-# pylint: disable=import-error
-REQUIREMENTS = []  # ['pyebox==0.1.0'] - disabled because it breaks pip10
+REQUIREMENTS = ['pyebox==1.1.2']
```

Comment:
Looking at the source for pyebox, I think timeouts are not done correctly. You're passing a `timeout` parameter to the `.get` and `.post` requests but that is not the overall timeout for the request. See this page for an example of timeouts in aiohttp: https://aiohttp.readthedocs.io/en/stable/